src/config.rs

```diff
@@ -2378,11 +2378,11 @@ impl GooseConfiguration {
         // Validate `no_metrics`.
         if self.no_metrics {
             // Status codes are not collected if metrics are disabled.
-            if self.status_codes {
+            if self.no_status_codes {
```

Comment:
Isn't this the other way around - status codes can't be enabled if we don't have metrics but can be disabled in any case?

Reply:
Thanks for catching that! This check is no longer relevant -- if `--no-metrics` is enabled then status codes are disabled automatically (whether or not `--no-status-codes` are enabled).